ui_framework/src/global_styling/reset/_reset.scss

```diff
@@ -1,6 +1,10 @@
-// Removes outline, but still provides a two color box shadow that works anywhere.
+// We apply brute force focus states to anything not coming from Eui
+// which has focus states designed at the component level.
 :focus {
-  @include focus;
```

Comment:
@cjcenizal Don't need you to review this whole PR, but can you think of any reason this approach for the focus stuff is a bad idea?